.github/dependabot.yml

```diff
@@ -7,6 +7,9 @@ updates:
       day: "monday"
       time: "14:00"
       timezone: "Asia/Tokyo"
+    open-pull-requests-limit: 5
+    reviewers:
```

Comment:
📝 
https://docs.github.com/ja/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#reviewers